lib/linearize.js

```diff
@@ -515,6 +518,22 @@ class BranchSnapshot {
     return this._rebuild(await this.autobase.latest())
   }
 
+  async updateSession (session) {
+    if (this.isRoot || !session._snapshotted) return this.update()
+    // If the session is a snapshot and this LinearizedCore is not the root, migrate the session to a clone of the root.
+    await this.root.update()
+    const rootClone = this.root.clone()
+    this._sessions.splice(session._sessionIdx, 1)
```

Comment:
this breaks all your other sessionIdx'es

Reply:
Ya fixed